3_visualize/src/visualize_data.R

```diff
@@ -0,0 +1,67 @@
+# Load necessary packages
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
```

Comment:
Again here, will be helpful to isolate which libraries are necessary for the function in this script.